db/models.py

```diff
@@ -50,3 +54,69 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
```

Comment:
Code Style Issue: It's a good practice to use `get_user_model()` instead of `settings.AUTH_USER_MODEL`. It's a good practice because it makes your code more flexible and less dependent on the settings.